packages/common/featureFlags.ts

```diff
@@ -0,0 +1,3 @@
+export interface FeatureFlags {
+  REACT_APP_EMAIL_FAX_OFFICIALS?: number
```

Comment:
1. This is meant to be treated as a bool.
2. Let's rename it to be `emailFaxOfficials` since it's not actually a flag anymore.
3. Let's make this field mandatory

Reply:
done